test/with_dummyserver/test_connectionpool.py

```diff
@@ -329,6 +329,7 @@ def test_socket_options(self):
         conn = pool._new_conn()
         conn.connect()
         s = conn._sock
+
```

Comment:
You still have this extra line. I'll merge the changes without this line, but please remove it. :)